__tests__/app.test.js

```diff
@@ -41,4 +42,26 @@ describe('app testing', () => {
             })
         })
     })
+    describe('get article by id', () => {
+        test('Returns the requested article', () => {
+            return request(app)
+            .get('/api/articles/2')
+            .expect(200)
+            .then(({ body: {article} }) => {
+                expect(article[0].article_id).toBe(2)
```

Comment:
Ideally we should be sending back a single object to the user not an object in an array. This can be amended in the Model 👍